src/Abstractions/NexusMods.Abstractions.DurableJobs/AOrchestration.cs

```diff
@@ -0,0 +1,56 @@
+namespace NexusMods.Abstractions.DurableJobs;
+
+/// <summary>
+/// A durable job that combines and orchestrates multiple sub jobs.
+/// </summary>
+public abstract class AOrchestration : IJob
+{ 
+    /// <summary>
+    /// Runs the orchestration. Calls in this method to start sub-jobs may pause the orchestration, and the orchestration may restart
+    /// many times before it finishes. All code in this method should be idempotent.
+    /// </summary>
+    internal abstract Task<object> Run(OrchestrationContext context, params object[] args);
```

Comment:
No, because it doesn't ever do anything that can be canceled, if this code blocks long enough to need a cancellation token it's not being used in the correct way